tests/OmniSharp.MiscellanousFiles.Tests/AssemblyInfo.cs

```diff
@@ -0,0 +1 @@
+﻿[assembly: Xunit.CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Why?

Reply:
It's in other tests projects as well. The problem is that we share the same MEF composition and OmniSharpWorkspace across a lot of tests. If tests run in parallel, unexpected things can happen.